docs/manual/java/guide/production/ProductionOverview.md

```diff
@@ -44,15 +76,15 @@ lagom.persistence.read-side.cassandra {
 }
 ```
 
-## Using static values for services and Cassandra to simulate a managed runtime
+## Using static values for services and Cassandra
```

Comment:
Now that I see the completed page, I think we're missing details on statically configuring Kafka here too. Above there's a bulletpoint that briefly mentions it but since we're adding a full section on statically configuring cassandra here, we shuold add one for Kafka or remove the Cassandra one.

Reply:
The Cassandra one has been there all along. I think it would be weird to delete it just for consistency.

The Kafka one was never written. I plan to do it, but I'd prefer to make that a separate pull request so that the changes for Lightbend Orchestration don't get delayed any longer. This has already expanded in scope far past what I intended. Unless you think it's unacceptable as is (net worse than what's on the site now) I'd rather get this merged and then continue to iterate in separate chunks.